website/docs/pushdown.md

```diff
@@ -52,3 +52,23 @@ Numbers outside the range of the safe IEEE 754 precision (`< -9007199254740991.0
 will prefetch all numbers larger/smaller than max/min value of the range.
 
 <!-- markdownlint-restore -->
+
+## Supported pushdown on `find()` cursor methods
+
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable MD001 MD033 MD051 -->
+
+|           | `limit()` | `skip()` | `sort()`                |
+| --------- | --------- | -------- | ----------------------- |
+| `limit()` | ✅        | ✖️       | ⚠️ <sub>[[2]](#2)</sub> |
+| `skip()`  | -         | ✖️       | ⚠️ <sub>[[3]](#3)</sub> |
+| `sort()`  | -         | -        | ⚠️ <sub>[[3]](#3)</sub> |
+
+###### [2] {#2}
+
+When cursor contains `limit()` and `sort()`, limit pushdown is applied
+only if experimental sort pushdown is enabled by `-enable-sort-pushdown` flag.
```

Comment:
That's not the right flag name.

Let's document `--test-disable-filter-pushdown` and `--test-enable-sort-pushdown` in the flags.md in a separate section with a warning

Reply:
Thanks, I added a section for pushdown query in the flags.md

Reply:
![image](https://github.com/FerretDB/FerretDB/assets/24355139/fd627fad-364c-45e1-a531-194fbfe54e09)